StaticAnalyzer/views/ios/static_analyzer.py

```diff
@@ -70,13 +72,19 @@ def static_analyzer_ios(request, api=False):
                 settings.UPLD_DIR, app_dict['md5_hash'] + '/')  # APP DIRECTORY
             tools_dir = os.path.join(
                 app_dict['directory'], 'StaticAnalyzer/tools/ios/')
+
+            # Will inject the different pattern strategy
+            # when it it will be requested
+            match_command = MatchCommand()
```

Comment:
Move to https://github.com/Arenash1313/Mobile-Security-Framework-MobSF/blob/master/StaticAnalyzer/views/ios/code_analysis.py#L34
and
https://github.com/Arenash1313/Mobile-Security-Framework-MobSF/blob/master/StaticAnalyzer/views/ios/binary_analysis.py#L66